bin/opencollective.js

```diff
@@ -0,0 +1,41 @@
+const child_process = require("child_process");
+const chalk = require("chalk");
+
+function padding(length) {
+	var padding = "";
+	for (var i = 0; i < length; i++) {
+		padding += " ";
+	}
+	return padding;
+}
+
+// Only show emoji on OSx (Windows shell doesn't like them that much ¯\_(ツ)_/¯ )
+function emoji(emoji) {
+	if (process.stdout.isTTY && process.platform === "darwin") {
+		return emoji;
+	} else {
+		return "";
+	}
+}
+
+function print(str, color) {
+	str = str || "";
+	color = color || "dim";
+	const terminalCols = process.platform === "win32" ? 80 : parseInt(child_process.execSync("tput cols").toString());
+	// eslint-disable-next-line no-control-regex
+	const strLength = str.replace(/\u001b\[[0-9]{2}m/g, "").length;
```

Comment:
could you store the regex into a variable and give to that variable a good name? So we know what's that regex for

Reply:
Makes sense :)